migration/i8.2z/postgresql/202112181000_IDEMPIERE-5110.sql

```diff
@@ -0,0 +1,10 @@
+-- Dic 18, 2021, 10:00:00 AM ART
+CREATE INDEX pp_product_bom_product_id_idx ON pp_product_bom (m_product_id ASC NULLS LAST)
```

Comment:
It is better to avoid the usage of "NULLS LAST" - we better stick to postgresql defaults in this case, that's another reason why is better to generate the index from the dictionary for consistency.